components/sections/app_promo/variant_a.tsx

```diff
@@ -21,125 +26,88 @@ function VariantA({ logo, subtitle, title, images }: AppPromoProps) {
-            <div className="mx-auto w-1/2 object-contain sm:hidden">
-              {images?.[currentPosition]?.image && (
-                <Image
-                  className="h-full w-full object-contain"
-                  src={urlFor(images?.[currentPosition]?.image)}
-                  sizes="(min-width: 520px) 224px, 45vw"
-                  width={500}
-                  height={850}
-                  alt={
-                    images?.[currentPosition]?.alt ??
-                    `appPromo-variantB-image${currentPosition}`
-                  }
-                />
-              )}
-            </div>
-            <div className="mb-16 flex justify-between sm:hidden">
-              <button
-                aria-label="Left Arrow button"
-                className="order-0 md:order-0 lg:order-0 inline-block rounded-full bg-white p-2 text-webriq-darkblue shadow hover:text-webriq-babyblue focus:outline-none xl:order-1 2xl:order-1"
-                onClick={arrowLeftClick}
-              >
-                <svg
-                  className="h-6 w-6"
-                  xmlns="http://www.w3.org/2000/svg"
-                  fill="none"
-                  viewBox="0 0 24 24"
-                  stroke="currentColor"
-                >
-                  <path
-                    strokeLinecap="round"
-                    strokeLinejoin="round"
-                    strokeWidth={2}
-                    d="M10 19l-7-7m0 0l7-7m-7 7h18"
-                  />
-                </svg>
-              </button>
-              <button
-                aria-label="Right Arrow button"
-                className="order-2 inline-block rounded-full bg-white p-2 text-webriq-darkblue shadow hover:text-webriq-babyblue focus:outline-none"
-                onClick={arrowRightClick}
-              >
-                <svg
-                  className="h-6 w-6"
-                  xmlns="http://www.w3.org/2000/svg"
-                  fill="none"
-                  viewBox="0 0 24 24"
-                  stroke="currentColor"
-                >
-                  <path
-                    strokeLinecap="round"
-                    strokeLinejoin="round"
-                    strokeWidth={2}
-                    d="M14 5l7 7m0 0l-7 7m7-7H3"
-                  />
-                </svg>
-              </button>
-            </div>
-          </div>
+    <section className="pt-16 overflow-hidden bg-primary">
+      <Container className="relative text-center" maxWidth={448}>
+        {logo?.image && (
+          <Link
+            aria-label={"Go to home page"}
+            className="inline-block p-5 mb-8 bg-white rounded-lg"
+            href={extractLink(logo)}
+          >
+            <Image
+              src={urlFor(logo?.image)}
+              width={50}
+              height={56}
+              alt={logo?.alt ?? "appPromo-logo"}
+            />
+          </Link>
+        )}
+        <Text className="mb-3 text-gray-50">{subtitle}</Text>
```

Comment:
Just like the logo image and other elements, kindly update the Heading and Text elements to be returned only when it is defined/there is a value.